app/controllers/workflows_controller.rb

```diff
@@ -33,28 +37,41 @@ def new
 
   def create
     skip_authorization
+    workflow_id = params[:workflow][:id]
 
-    workflow_hash = credential.accessible_workflow?(params[:workflow][:id])
+    workflow_hash = { id: workflow_id }
+    workflow_hash = credential.accessible_workflow?(params[:workflow][:id]) unless Rails.env.development?
 
     unless workflow_hash.present?
       skip_authorization
       head :forbidden
       return
     end
 
-    @workflow = Workflow.new(workflow_params.merge(id: params[:workflow][:id],
-                                                   project_id: workflow_hash["links"]["project"]))
+    panoptes_workflow = Effects.panoptes.workflow(workflow_id) unless Rails.env.development?
```

Comment:
I think the display_name should already be in that workflow_hash?

Reply:
It wasn't clear to me whether the workflow hash would contain that information if this were a new workflow that the user hadn't seen--where is the code that does that?